src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -659,6 +659,19 @@ enquote = do
      then singleQuoted <$> withQuoteContext InSingleQuote tok
      else doubleQuoted <$> withQuoteContext InDoubleQuote tok
 
+doAcronym :: PandocMonad m => String -> LP m Inlines
+doAcronym form = do
+  acro <- braced
+  return . mconcat $ [spanWith ("",[],[("acronym-label", toksToString acro), ("acronym-form", "singular+" ++ form)]) 
```

Comment:
This line, and 672, should be wrapped to stay within 80 columns, as mentioned in CONTRIBUTING.md.